sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -1239,4 +1242,32 @@ class CachedTableSuite extends QueryTest with SQLTestUtils
       }
     }
   }
+
+  test("SPARK-33729: REFRESH TABLE should not use cached/stale plan") {
+    def mf(src: File, dst: File): Unit = {
```

Comment:
If you don't mind, shall we use a more longer descriptive name?

Reply:
sure - let me figure out one and update

Reply:
Thanks!